zjit/src/state.rs

```diff
@@ -136,6 +136,34 @@ impl ZJITState {
     pub fn get_counters() -> &'static mut Counters {
         &mut ZJITState::get_instance().counters
     }
+
+    pub fn should_save_compiled_iseqs() -> bool {
+        ZJITState::get_instance().options.save_compiled_iseqs.is_some()
+    }
+
+    pub fn log_compile(iseq_name: String) {
```

Comment:
Could you add a `///` comment for this function? This is a fairly complicated logic, so it's not immediately clear what it's doing from the function name or staring at the definition.